test/profiler.js

```diff
@@ -21,6 +23,14 @@ describe('profiler', function() {
     });
   });
 
+  after("Cleanup tmp files", function(done){
+    glob('tmp-*', (err, files) => {
+      if(err) done(err);
+      files.forEach(file => fs.removeSync(file));
+      done();
+    })
+  })
```

Comment:
There's gotta be a better way to make sure those get cleaned up. Well, whatev.

Reply:
Add it the npm script? ` && rm -rf tmp-*`

Reply:
It's probably not worth the change right now, but I like that idea